packages/plugin-session-replay-browser/src/session-replay.ts

```diff
@@ -103,9 +118,23 @@ class SessionReplay implements SessionReplayEnrichmentPlugin {
         void this.storeEventsForSession(this.events, this.currentSequenceId);
       },
       packFn: pack,
+      maskAllInputs: true,
     });
   }
 
+  shouldSplitEventsList = (nextEventString: string): boolean => {
```

Comment:
Can we add a comment to what the purpose of this function is?

Reply:
Reading this - it seems to be controlling the batching logic?

Reply:
Yeah thats right